compose_to_rancher/__init__.py

```diff
@@ -6,20 +6,34 @@
 default_file_path = 'docker-compose.yml'
 default_output_file_path = 'docker-compose-v1.yml'
 
+EXCLUDED_OPTIONS = ['build', 'depends_on']
+
+#######################
+# Base Functions
+#######################
 
 def convert_compose_to_rancher(compose_v2_data):
     """
     Convert a parsed yaml data structure docker-compose V2 to a \
     V1 docker-compose data structure
     """
     new_compose = {}
-    for service, contents in compose_v2_data['services'].items():
+    for service, service_options in compose_v2_data['services'].items():
+        #error if no image option is declared
+        if 'image' not in service_options:
+            raise ValueError('no image in service {}'.format(option))
```

Comment:
@tiangolo 
You are right, I think this problem is also present on the current version. Look at the version on master:
https://github.com/tiangolo/compose-to-rancher/blob/master/compose_to_rancher/__init__.py#L22

I'll change it right away.